src/app/qgisapp.cpp

```diff
@@ -5940,6 +5959,72 @@ QList<QgsMapLayer *> QgisApp::askUserForOGRSublayers( QgsVectorLayer *layer )
   return result;
 }
 
+QList<QgsMapLayer *> QgisApp::askUserForMDALSublayers( QgsMeshLayer *layer )
```

Comment:
cannot we send directly subLayers string to this function and assume it is not empty?

Reply:
maybe, but like this, it is consistent with same method for other layer type, and in the method we need the name of the parent layer if group creation is asked.

Reply:
ok